jenkins/pipelines/cd/nightly/build-darwin-amd64-4.0.groovy

```diff
@@ -41,6 +41,8 @@ def githash
 def os = "darwin"
 def arch = "amd64"
 def tag
+def taskStartTimeInMillis = System.currentTimeMillis()
+def tiflash_result = "NOT TRIAGED"
```

Comment:
What does "not triaged" mean?

Reply:
Sorry, I mean 'triggered'..

This pipeline will compile all the components instead of only TiFlash, for example:
https://cd.pingcap.net/blue/organizations/jenkins/build-darwin-amd64-4.0/detail/build-darwin-amd64-4.0/59/pipeline
![image](https://user-images.githubusercontent.com/30543181/123934960-5eabc700-d9c6-11eb-9fb8-9717ff3e0fd9.png)

but if some components compile failed, the pipeline will fail immediately and will not trigger subsequent component compilation, like this :
https://cd.pingcap.net/blue/organizations/jenkins/build-darwin-amd64-4.0/detail/build-darwin-amd64-4.0/62/pipeline/10/
![image](https://user-images.githubusercontent.com/30543181/123935278-a894ad00-d9c6-11eb-9fb2-884f9128e5c8.png)

so I add a status to express this situation or when 'skip-tiflash' variable is set.